### DISCUSSION TITLE
石头自清洁扫拖机器人G10S相关问题

### DISCUSSION THREAD

**Discussion by xzshow** [XiaoMi/ha_xiaomi_home#145] [Ideas]
石头自清洁扫拖机器人G10S，将房间分区后，能不能将单个房间按钮接入到HA中，比如接入到HA中是客厅按钮，按下后扫地机就打扫客厅，谢谢

**Comment by JimmyKmi** [comment]
可参考

https://web.vip.miui.com/page/info/mio/mio/detail?isTop=0&postId=41565796&fromBoardId=&fromPage=mioHomePage&fromPathname=mioHomePage&app_version=dev.230112

https://github.com/XiaoMi/ha_xiaomi_home/discussions/109

**Comment by xzshow** [comment]
关键没用极客，小米民间的别人也搞了，这要我们自己研究，唉



发自我的iPhone


------------------ 原始邮件 ------------------
发件人: JimmyKmi ***@***.***&gt;
发送时间: 2024年12月18日 15:09
收件人: XiaoMi/ha_xiaomi_home ***@***.***&gt;
抄送: xzshow ***@***.***&gt;, Author ***@***.***&gt;
主题: Re: [XiaoMi/ha_xiaomi_home] 石头自清洁扫拖机器人G10S相关问题 (Discussion #145)





 
可参考
 
https://web.vip.miui.com/page/info/mio/mio/detail?isTop=0&amp;postId=41565796&amp;fromBoardId=&amp;fromPage=mioHomePage&amp;fromPathname=mioHomePage&amp;app_version=dev.230112
 
#109
